src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,27 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+export const PostInfo = ({ post }) => {
+  return (
+    <>
+      <div className="PostInfo">
+        <div className="PostInfo__header">
+          <h3 className="PostInfo__title">{post.title}</h3>
+
+          <p>
+            {' Posted by  '}
+
+            <UserInfo user={post.user} />
```

Comment:
user can be null, so let's add the check here
```suggestion
           {post.user && <UserInfo user={post.user} />}
```